lib/package.rb

```diff
@@ -157,6 +159,15 @@ def self.#{prop}
     end
   end
 
+  def self.compatible?
+    if @compatibility
+      return (@compatibility.casecmp?('all') or @compatibility.include?(ARCH))
+    else
+      warn "#{name}: Missing `compatibility` field.".yellow
+      return true
```

Comment:
`return false`?  We do not want to continue if `@compatibility` is not set or false, right?

Reply:
Changed to `abort`